src/Sylius/Bundle/CoreBundle/Doctrine/ORM/PaymentMethodRepository.php

```diff
@@ -24,7 +24,6 @@ public function createListQueryBuilder($locale)
     {
         return $this
             ->createQueryBuilder('o')
-            ->addSelect('translation')
```

Comment:
I am pretty sure this will cause to lazy load translation contents anyway, as they will not be part of the select clause.